GoogleForCreators/web-stories-wp
# Editor: Autoset media as bg if user scales it to fit the page

## Summary



## Relevant Technical Choices



## To-do

- [x] Snackbar message

## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

Every time the media element (image, video, gif) will be resized (or moved) so that it covers the whole canvas - it will be set as background media.
If there is already background media on a page - it will not be set.

1. Add image
2. Resize it to cover the canvas
3. Should be auto-set as the background image


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #5356
